### PR TITLE
Lpc flash support

### PR DESCRIPTION
**Pull request by c1728p9**
Add flash support to the lpc11u35 interface so it can store settings.

### CODE REVIEW

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by c1728p9** [comment]
@mbed-bot: TEST

File: source/daplink/daplink.sct [hunk 1]
```diff
@@ -33,11 +33,13 @@ LR_IROM1 DAPLINK_ROM_APP_START DAPLINK_ROM_APP_SIZE {
   RW_IRAM1 DAPLINK_RAM_APP_START DAPLINK_RAM_APP_SIZE {  ; RW data
    .ANY (ram_func)
    .ANY (+RW +ZI)
+   .ANY (RAM1)
   }
 
   #if defined(DAPLINK_RAM_APP2_START)
       RW_IRAM2 DAPLINK_RAM_APP2_START DAPLINK_RAM_APP2_SIZE {  ; RW data
        .ANY (+RW +ZI)
+	   .ANY (RAM2)
```
Comment (sg-) [inline source/daplink/daplink.sct]: nit: extra tab

**Comment by c1728p9** [comment]
The last 32 bytes of RAM can be clobbered by the IAP calls. I removed this region from the memory map [here](https://github.com/mbedmicro/DAPLink/pull/255/files#diff-8d605fbb650e7188d924bb38e686e453R34) to prevent problems.

File: source/daplink/daplink.sct (hunk 1 above)
Comment (c1728p9) [inline source/daplink/daplink.sct]: Good catch. Fixed

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Pull request merged**